WORKSPACE

```diff
@@ -7,3 +10,46 @@ http_archive(
 load("@io_bazel_rules_go//go:def.bzl", "go_repositories", "go_repository")
 
 go_repositories()
+
+#=============================================================================
+# Docker rules
+
+git_repository(
+    name = "io_bazel_rules_docker",
+    remote = "https://github.com/bazelbuild/rules_docker.git",
+    tag = "v0.3.0",
+)
+
+load(
+    "@io_bazel_rules_docker//container:container.bzl",
+    "container_pull",
+    container_repositories = "repositories",
+)
+
+container_repositories()
+
+container_pull(
+  name = "debian_hyperkube_base_amd64",
+  registry = "gcr.io",
+  repository = "google_containers/debian-hyperkube-base-amd64",
```

Comment:
Why we using hypercube base?

Reply:
Shouldn't we just use the base Debian??

Reply:
dns-controller currently uses `gcr.io/google_containers/debian-base-amd64:0.1`.  We need ca-certificates to talk to the EC2 API.  We can't install ca-certificates using bazel because installing ca-certificates requires code execution.  It looks like google has some work going on in https://github.com/GoogleCloudPlatform/distroless and https://github.com/GoogleCloudPlatform/debian-docker/tree/master/reproducible that will eventually fix this, but in the meantime debian_hyperkube_base_amd64 is based on debian-base-amd64, but adds ca-certificates.

It also adds lots of other stuff we don't really want, but on the flipside it is more likely to be useful on the nodes, given its status as the de-facto k8s base image.